### PR TITLE
Marines automation

### PR DESCRIPTION
**Pull request by 88mary256**

### CODE REVIEW

File: src/marines/automation/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Instalar requests con pip
+# pip install requests
+# if pip no es reconocido en cmd añadir en variables de entorno:
```
Comment (sperBruno) [inline src/marines/automation/__init__.py]: delete this

File: src/marines/automation/create_project.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import json
+import requests
+import pprint
+
+#class AutomationForTodoLy():
+
+host = "https://todo.ly/api/"
+#def create_project():
+endpoint = host + "projects.json"
+header = {"Authorization": "Basic Z2F0aXRhMDEwMS5teG1AZ21haWwuY29tOjEyMzQ1Njc4",
```
Comment (sperBruno) [inline src/marines/automation/create_project.py]: do not use credentials or do not push credentials

File: src/marines/automation/create_project.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+ u'ProjectShareOwnerEmail': None,
+ u'ProjectShareOwnerName': None,
+ u'SyncClientCreationId': None}
+3782188
+'''
+
+'''
+tarea crear read update delete
+
+hacer 1 proyecto con su subproyecto y una tarea en el subprojecto
```
Comment (sperBruno) [inline src/marines/automation/create_project.py]: delete  commented lines

File: src/marines/automation/project.py [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+def main():
+    id = create("New project")
+    readAll()
+    read(id)
+    update(id,"a different project name")
+    delete(id)
+    print "finish project CRUD"
+
+
+main()
```
Comment (sperBruno) [inline src/marines/automation/project.py]: use unit test instead

File: src/marines/automation/project.py [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+              "Content-Type": "application/json"}
+    # name = "My 2nd project"
+    body = {"Content": name}
+    print "Creating project with name " + name
+    response = requests.post(url=endpoint, data=json.dumps(body), headers=header)
+    response_json = response.json()
+    print("Create Project status code: %s" % response.status_code)
+    pprint.pprint(response.json())
+
+    assert response.status_code == 200, "Failed"
```
Comment (sperBruno) [inline src/marines/automation/project.py]: separate unit test from functional code

**Review by sperBruno — CHANGES_REQUESTED** [review]

File: src/marines/automation/create_project.py (hunk 2 above)
Comment (88mary256) [reply src/marines/automation/create_project.py]: this is only class example

File: src/marines/automation/__init__.py (hunk 1 above)
Comment (88mary256) [reply src/marines/automation/__init__.py]: fixed

File: src/marines/automation/create_project.py (hunk 2 above)
Comment (88mary256) [reply src/marines/automation/create_project.py]: fixed

File: src/marines/automation/create_project.py (hunk 3 above)
Comment (88mary256) [reply src/marines/automation/create_project.py]: fixed

File: src/marines/automation/project.py (hunk 4 above)
Comment (88mary256) [reply src/marines/automation/project.py]: fixed

File: src/marines/automation/project.py (hunk 5 above)
Comment (88mary256) [reply src/marines/automation/project.py]: fixed

**Pull request merged**